model/Image.php

```diff
@@ -70,7 +70,7 @@ class Image extends File implements Flushable {
 	 * @config
 	 * @var bool Force all images to resample in all cases
 	 */
-	private static $force_resample = false;
+	private static $force_resample = true;
```

Comment:
This is my highly opinionated opinion, but this essentially makes all size checks to PREVENT unnecessary respampling essentially voided. I understand Hamish's point that clients could upload unoptimised graphics, but they could also upload perfectly optimised graphics that deteriorate through respampling. It essentially undoes a lot of the work that was done in the past to prevent this happening.

See https://github.com/silverstripe/silverstripe-framework/pull/1318

I would prefer this was off by default (and stayed off).